lib/landing/src/Wrapper.tsx

```diff
@@ -60,20 +61,13 @@ export default function Wrapper({ title, page, children }: WrapperProps): ReactE
             }}
           >
             <Box sx={{ display: { xs: 'flex', cursor: 'pointer' } }}>
-              <Link href='/' color='inherit' underline='none'>
-                <Image
-                  loader={imageLoader}
-                  src='/logo-horizontal-light.png'
-                  alt='Logo'
-                  width={142}
-                  height={60}
-                  priority
-                />
+              <Link href='/'>
+                <Image loader={imageLoader} src={bailoLogo} alt='Logo' width={142} height={60} priority />
               </Link>
             </Box>
-            <Button href='/docs/' sx={{ color: 'white' }}>
-              Documentation
-            </Button>
+            <Link href='/docs'>
+              <Button sx={{ color: 'white' }}>Documentation</Button>
+            </Link>
             <Button href='https://github.com/gchq/bailo' sx={{ color: 'white' }}>
```

Comment:
If we're making the change to the documentation button we should replicate it here too and wrap this button in `<Link />`